src/tools/miri/tests/pass/intrinsics.rs

```diff
@@ -33,6 +33,21 @@ fn main() {
     assert_eq!(intrinsics::likely(false), false);
     assert_eq!(intrinsics::unlikely(true), true);
 
+    let mut saw_true = false;
+    let mut saw_false = false;
+
```

Comment:
```suggestion
    let mut saw_false = false;
```
just a not to make it visually more clear that these are all together testing one thing